fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -1043,6 +1043,7 @@ private void receiveInteraction(VehicleSensorActivation vehicleSensorActivation)
      */
     private void receiveInteraction(VehicleParametersChange vehicleParametersChange) throws InternalFederateException {
         if (externalVehicles.containsKey(vehicleParametersChange.getVehicleId())) {
+            changeExternalParameters(vehicleParametersChange);
```

Comment:
Probably by accident. Will revert

Reply:
I don't remember putting anything there though, very interesting. Can it be that this is a relic of one of your commits that I somehow have left over because of git actions?

Reply:
Oooh, git blame says I added this sometime in June. Maybe I missed it in a commit then and it somehow ended up here? But I wouldn't know how exactly.

Reply:
I found out what it was. It was for the AvH workshop, a change that allowed changing colors of extenally added vehicles. I am unsure if we should still integrate this in some shape or form